src/app/page/Error/ErrorLanding.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+import MuiContainer from "@material-ui/core/Container";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import { Page } from "../../layout";
+import PageNotFound from "./PageNotFound";
+import Unauthorized from "./Unauthorized";
+
+const errorLandingStyles = makeStyles((theme) => ({
+  errorContent: {
+    padding: "1rem",
+  },
+}));
+/**
+ * Component for Error landing page
+ *
+ * @component
+ */
+const ErrorLanding = ({ errorCode }) => {
+  const classes = errorLandingStyles();
+
+  return (
+    <Page className="error-page">
+      <MuiContainer className={clsx("error-page-content", classes.errorContent)}>
```

Comment:
are we moving toward a css based system as well ?

Reply:
i've just added it in case we need it, and also helps with semantics. The styling itself is all still done via React styles currently